app/services/authentication/providers/twitter.rb

```diff
@@ -28,6 +28,10 @@ def existing_user_data
         }
       end
 
+      def get_email
+        info.email.to_s
+      end
```

Comment:
This method has to be in this class because `info` is `protected`, and can't be in called in the `Authenticator` class.

Reply:
I think we should promote this to the base class, as it'd be weird to have a method that's only available on one provider, we coul also just call it `user_email` similar to:

```ruby
      def user_nickname
        info.nickname.to_s
      end
```

you can find in the `Provider` base class